nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/utils/AzureStorageUtils.java

```diff
@@ -98,8 +102,25 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
-    // use HTTPS by default as per MSFT recommendation
+    public static final PropertyDescriptor STORAGE_CREDENTIALS_SERVICE = new PropertyDescriptor.Builder()
+            .name("storage-credentials-service")
+            .displayName("Storage Credentials")
+            .description("The Controller Service used to obtain Azure Storage Credentials. The credentials can be configured through a common/shared controller service " +
+                    "instead of the processor level properties.")
+            .identifiesControllerService(AzureStorageCredentialsService.class)
+            .required(false)
+            .build();
+
+    /**
+     * @deprecated Not used, will be removed.
+     */
+    @Deprecated
     public static final String FORMAT_BLOB_CONNECTION_STRING = "DefaultEndpointsProtocol=https;AccountName=%s;AccountKey=%s";
+
+    /**
+     * @deprecated Not used, will be removed.
+     */
+    @Deprecated
```

Comment:
Instead of marking these unused constants deprecated, I think it's safe to just remove them.